README.md

```diff
@@ -86,13 +86,107 @@ The CouchDBSample executable demonstrates how to create, read, update and delete
     .build/debug/CouchDBSample
     ```
 
-    You will see informational messages such as "Successfully created the following JSON document in CouchDB:" for each of the operations (create, read, update and delete) performed on the `kitura_test_db` database.
+    You should see informational messages such as "Successfully created the following JSON document in CouchDB:" for each of the operations (create, read, update and delete) performed on the `kitura_test_db` database.
 
-## Example
+## API Documentation
 
-For a more comprehensive example, you can follow the Kitura tutorial [Getting Started with Server-side Swift on raywenderlich.com](https://www.raywenderlich.com/180721/kitura-tutorial-getting-started-with-server-side-swift) that shows you how to create a backend API and then link this to a CouchDB instance running on your local machine.
+#### Document
+
+CouchDB is a NoSQL database for storing documents. A `Document` is any structure that can be represented as JSON and contains `_id` and `_rev` fields.  
+The `_id` field is the unique identifier for the document. If it is not set, a random UUID will be assigned for the document.  
+The `_rev` field is the revision of the document. It is returned when you make requests and is used to prevent conflicts from multiple users updating the same document.  
```

Comment:
Andy, can you bullet point or similar the definitions for _id field and _rev field (or put them in a separate paragraph)?